plugins/CorePluginsAdmin/PluginInstaller.php

```diff
@@ -297,19 +297,16 @@ private function fixPluginFolderIfNeeded($tmpPluginFolder)
 
     private function copyPluginToDestination($tmpPluginFolder)
     {
-        // if MATOMO_PLUGIN_COPY_DIR is defined , copy plugin to this directory too
+        $pluginsDir = Manager::getPluginsDirectory();
 
-        $pluginsDirs = [Manager::getPluginsDirectory()];
-        
         if (!empty($GLOBALS['MATOMO_PLUGIN_COPY_DIR'])) {
```

Comment:
In case someone configures `$GLOBALS['MATOMO_PLUGIN_COPY_DIR']` directly, do we maybe also need to check that directory is actually one of the configured plugin directories?

Reply:
i made a change in `initPluginDirectories` to check `$GLOBALS['MATOMO_PLUGIN_COPY_DIR']` in both cases ( via env var or directly )